assets/js/dailies.js

```diff
@@ -216,8 +246,12 @@ class SynchronizeDailies {
       }
     })();
 
-    if ($(`[data-text="${this.key}"]`).parent().hasClass('disabled') ||
-      $(`[data-text="${this.key}"]`).parent().parent().hasClass('disabled'))
-      $(`[data-text="${this.key}"]`).trigger('click');
+    const targetEl = document.querySelector(`[data-text="${this.key}"]`);
+    if (
+      targetEl.parentElement.classList.contains('disabled') ||
+      targetEl.parentElement.parentElement.classList.contains('disabled')
+    ) {
+      targetEl.click();
+    }
```

Comment:
It needs some error handler here. It was my oversight, that key `"menu.cmpndm.animal_ram"` does not exists. Today throws an error and doesn't show anything on the map. Earlier error was thrown silently.
And of course update "ram" => "bighornram_desert" in possible_dailies.json